### DISCUSSION TITLE
How to enroll GitHub copilot with student card

### DISCUSSION THREAD

**Discussion by zoecheng0902** [community/community#59710] [GitHub Education]
### Hi everyone,

Hi everyone,
I am Taiwanese student and I've tried many times to enroll GitHub copilot with student card but unfortunately, I have got the reject message many times, so could anyone can teach or tell me how to register with GitHub and GitHub copilot.

PS: Could anyone can give me user guideline ,thanks

### A suggestion...(optional)

How to enroll GitHub copilot with student card

**Comment by samus-aran** [comment]
Hi @zoecheng0902 👋 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
